src/overlays/actors/ovl_En_Ani/z_en_ani.c

```diff
@@ -1,21 +1,44 @@
 /*
  * File: z_en_ani.c
  * Overlay: ovl_En_Ani
- * Description: Part-time worker
+ * Description: Man in Tree in South Termina Field
+ *    version in shop at night is EnOssan actor using object_ani
  */
 
 #include "z_en_ani.h"
 
-#define FLAGS 0x00000009
+#define FLAGS (ACTOR_FLAG_8 | ACTOR_FLAG_1)
 
 #define THIS ((EnAni*)thisx)
 
+// clang-format off
+#define ANI_STATE_STANDING  (0)
+#define ANI_STATE_UNK       (1 <<  0)
+#define ANI_STATE_SUFFERING (1 <<  1)
+#define ANI_STATE_CLIMBING  (1 <<  2)
+#define ANI_STATE_FALLING   (1 <<  3)
+// clang-format on
```

Comment:
Does this really need clang-format off?

Reply:
Need? No.
This is to keep the visual alignment of the values only, as without it formatter shifts all values to single white space spacing, making it harder to read

I do see I need to remove a space after the shift operator though.. not sure why that extra space is there.

Reply:
I don't think the irregular spacing is a huge deal, and I think we should try to minimize the usage of `clang-format off`

Reply:
It really bugs me, more than the clang formater off syntax does for some reason